FreeRTOS/FreeRTOS
# Add option to disable trace on enter

Add option to disable trace on enter

Description
-----------
Trace on Enter is causing a testing job to fail

Test Steps
-----------
Local testing
```
make clean && make USER_DEMO='FULL_DEMO'   TRACE_ON_ENTER='0'
```
No trace on enter message:
```
[Posix_GCC]$ ./build/posix_demo

Trace started.
The trace will be dumped to disk if a call to configASSERT() fails.
Starting full demo
```
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
